src/fungible_token.rs

```diff
@@ -406,17 +406,14 @@ impl FungibleToken {
     }
 
     pub fn storage_withdraw(&mut self, amount: Option<u128>) -> StorageBalance {
-        let predecessor_account_id_bytes = sdk::predecessor_account_id();
-        let predecessor_account_id = str_from_slice(&predecessor_account_id_bytes);
-        if let Some(storage_balance) = self.internal_storage_balance_of(predecessor_account_id) {
-            match amount {
-                Some(amount) if amount > 0 => {
-                    sdk::panic_utf8(b"ERR_WRONG_AMOUNT");
-                }
-                _ => storage_balance,
+        let acc_id = sdk::predecessor_account_id();
+        let predecessor_account_id = str_from_slice(&acc_id);
+        let storage_balance = self.internal_storage_balance_of(predecessor_account_id);
+        match amount {
+            Some(amount) if amount > 0 => {
+                sdk::panic_utf8(b"ERR_WRONG_AMOUNT");
             }
-        } else {
-            sdk::panic_utf8(b"ERR_ACCOUNT_NOT_REGISTERED");
+            _ => storage_balance,
```

Comment:
The previous implementation was correct, as according to `storage_withdraw` specs in NEP-145:
> Panics:
If the account is not registered with the contract
If the specified withdrawal amount is greater than the account's available storage balance

So this part should be reverted.